### PR TITLE
Background v2

### PR DESCRIPTION
**Pull request by Ajver**
Add background images, bushes, path, and so on. Make it moving as paralax background + fix some bugs I found while writing better background

### CODE REVIEW

File: Scenes/World/Layer.gd [hunk 1]
```diff
@@ -4,7 +4,7 @@ var TREE_TEXTURES = [
 	load("res://Objects/Background/Trees/tree1.png"),
 	load("res://Objects/Background/Trees/tree2.png"),
 	load("res://Objects/Background/Trees/tree3.png"),
-	load("res://Objects/Background/Trees/tree4.png"),
+	#load("res://Objects/Background/Trees/tree4.png"),
```
Comment (B0ZIA) [inline Scenes/World/Layer.gd]: Why the tree4 is worse than the rest?

File: Scenes/World/Layer.gd [hunk 2]
```diff
@@ -46,11 +46,11 @@ func generate_tree() -> void:
 	add_child(tree)
 	
 	tree.position.x = -position.x + WINDOW_WIDTH + 200
-	tree.position.y = 550
+	tree.position.y = 520
```
Comment (B0ZIA) [inline Scenes/World/Layer.gd]: Why 520? Maybe:
```tree.position.y = TREE_EXTENT```
and above in the code to give:
```const TREE_EXTENT= 520```

File: Scenes/World/Layer.gd (hunk 1 above)
Comment (Ajver) [reply Scenes/World/Layer.gd]: We was talking about this in the Monday's meeting and we decited to remove that tree, becouse it's not matches other. I think I should remove that line. Do I?

File: Scenes/World/Layer.gd (hunk 2 above)
Comment (Ajver) [reply Scenes/World/Layer.gd]: You are really right. It's not good practice to hard coding.

File: Scenes/World/Layer.gd (hunk 1 above)
Comment (B0ZIA) [reply Scenes/World/Layer.gd]: Yes, please

**Comment by Ajver** [comment]
I fixed conflicts and it's ready to merge

**Pull request merged**